Settings.hs

```diff
@@ -80,6 +83,7 @@ instance FromJSON AppSettings where
         appCopyright              <- o .: "copyright"
         appAnalytics              <- o .:? "analytics"
 
+        appDevelopment            <- o .:? "development"      .!= defaultDev
```

Comment:
We've specifically been avoiding a general "development" setting, instead having specific settings like detailed-logging and mutable-static.